resources/dicts/thoughts/alive/general.json

```
@@ -51,87 +49,1424 @@
             "Is craving the taste of fish",
             "Is wondering if {PRONOUN/m_c/subject} would be a good swimmer",
             "Is feeling underappreciated...",
+            "Woke up on the wrong side of the nest",
             "Is staring off into space",
             "Is picking the burrs from {PRONOUN/m_c/poss} pelt",
-            "Is sharpening {PRONOUN/m_c/poss} claws",
-            "Woke up on the wrong side of the nest",
-            "Can't wait to take a nap later",
             "Is batting at {PRONOUN/m_c/poss} tail",
             "Daydreams about being a warrior in LionClan",
             "Daydreams about being a warrior in TigerClan",
             "Daydreams about being a warrior in LeopardClan",
-            "Saw a yappy little dog today",
-            "Got startled by a squirrel",
             "Spends time watching fish in the nearby river",
             "Is batting at a beam of sunlight",
+            "Didn't get enough sleep",
+            "Is stretching out",
+            "Is watching cats come and go from camp",
             "Thinks about StarClan",
+            "Is watching the sky for hawks",
+            "Is watching the sky for rain",
+            "Is cloudwatching",
+            "Is debating on which prey to take from the fresh-kill pile",
+            "Sits quietly watching cats go about their day",
+            "Is wondering if there are clans outside of the ones around {PRONOUN/m_c/poss} area",
```

```suggestion
            "Is wondering if there are Clans other than the ones around {PRONOUN/m_c/poss} area",
```
changing "outside of" to "other than" is a minor nitpick